### ISSUE TITLE
Infinite Slider navigation issues

### ISSUE THREAD

**Issue by alexiglesias93** [finsweet/webflow-addons#1]
Made a video showing weird behaviors when changing slides on reverse and when clicking the slider dots:
https://www.loom.com/share/7877c14425af4506aae02c0cceb452a3

**Comment by alexiglesias93** [comment]
@Xlvis great process so far. One thing that you're missing in your implementation is keyboard navigation.
I see you're listening for click events on the slider arrows and dots, but users can also navigate through slides using the arrow keys.
Try it our yourself to understand what I mean.

Also, the sliders have an option of `auto-play`, which continuously switches slides after X milliseconds. This, again, is not triggered with click events.

I am not a fan of relying just on click events, as you can see, you miss many scenarios where those are not even fired. I would suggest you try to keep track of the currently active slide in a different way, perhaps you could try again using a `MutationObserver`?

**Comment by xlvisben** [comment]
@alexiglesias93 I haven't managed to get the slider to work with `arrow keys` how is it done. I have tried hitting `tab key` but it hasn't been succesful

**Comment by alexiglesias93** [comment]
@Xlvis https://www.loom.com/share/9900d95ff342427d9a08f031e7718c48
